backend/src/services/worker/worker.service.ts

```diff
@@ -233,13 +242,13 @@ class Worker {
     await Promise.all(
       spaces.map((item) => {
         const id = item.id;
-        return this.#spaceService.delete({ id, token });
+        return this.#spaceService.delete({ id });
       }),
     );
     await Promise.all(
       slcFunctions.map((slcFunction) => {
         const id = slcFunction.id;
-        return this.#slcFunctionService.delete({ id, token });
+        return this.#slcFunctionService.delete({ id });
```

Comment:
I don't like this solution, honestly. It is not obvious why token in SlcFunctionService.delete can be null. You have 2 possible options:
1. Do not use slcFunctionService.delete and spaceService.delete. And just copy code from these methods into WorkerService.deleteWorker. (can't recommend this approach)
2. Move role checks to the api layer (as with permissions check)